README.md

```diff
@@ -30,7 +35,17 @@ Your Answer:
 
 ```js
 // your code here
+function Instructor(name){
+
+this.name = name
+givesHomework = function(assignment){
+  console.log(name , assignment);
+}  
+}
+
+Instructor.prototype.Robin = function givesHomework(Intro to Ruby){
 
```

Comment:
this would define a method called `Robin` on the `instructor` prototype, we've already done something like that when we defined our method `givesHomework` in the constructor definition